server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
const express = require("express");
```